docs/integrations/containers-orchestration/opentelemetry/vmware-opentelemetry.md

```diff
@@ -0,0 +1,254 @@
+---
+id: vmware-opentelemetry
+title: VMWare - OpenTelemetry Collector
+sidebar_label: VMWare - OTel Collector
+description: Learn about the Sumo Logic OpenTelemetry app for VMWare.
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+<img src={useBaseUrl('img/integrations/containers-orchestration/vmware.png')} alt="VMware dashboards" width="50" />
+
+The VMware - OpenTelemetry app uses metrics from the VMware cloud computing virtualisation platform to enable monitoring of vCenter, datacenter, resource pool, cluster, ESXi hosts and individual virtual machines metrics with real-time date displayed in predefined dashboards.
+
+The dashboards provide insight into key metrics such as CPU, memory, disk utilization at different infrastructure level like vm, host, datacenter, resource pool and clusters. This enables you to determine capacity constraints and troubleshoot operational issues related to over-provisioning, changes to configuration, and VM movement.
+
+VMWare metrics are collected through the [vCenter Receiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/vcenterreceiver) of OpenTelemetry.
+
+See the [vSphere product page](https://www.vmware.com/products/vsphere.html) for more information on VMware hybrid cloud.
+
+<img src='https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/VMWare-OpenTelemetry/VMWare-Schematics.png' alt="Schematics" />
```

Comment:
This image does not load in my local build. When I go to the URL, it says "AccessDenied".

Reply:
Uploaded to S3 and made public. No changes required in the PR.